src/snowflake/snowpark/functions.py

```diff
@@ -6702,17 +6702,75 @@ def sproc(
         registered. Invoking :func:`sproc` with ``replace`` set to ``True`` will overwrite the
         previously registered function.
 
+        4. To describe the return type for a table stored procedure, use
+        either of the following ways:
```

Comment:
"either" suggests that there are only 2 options and it looks like there are 3 ways to do this.
I suggest that you change "either" to "one".